src/lib/booleans.js

```diff
@@ -0,0 +1,88 @@
+function negate(a) {
+  return !a;
+};
+
+function both(a, b) {
+  return a && b;
+};
+
+function either(a, b) {
+  return a || b;
+};
+
+function none(a, b) {
+  return !a && !b;
+};
+
+function one(a, b) {
+  return (a && !b) || (!a && b);
+}
+
+function truthiness(a) {
+  return !!a;
+};
+
+function isEqual(a, b) {
+  return a === b;
+};
+
+function isGreaterThan(a, b) {
+  return a > b;
+};
+
+function isLessThanOrEqualTo(a, b) {
+  return a <= b;
+};
+
+function isOdd(a) {
+  return a % 2 !== 0;
+};
+
+function isEven(a) {
+  return a % 2 == 0;
```

Comment:
Would recommend to always use === instead of ==, as === checks the type as well.